nss-day-cohort-41/tabloid-fullstack-loch-ness-cryptozoologists
# Jrr Category #9 List

Issue #9 View All Categories:
Upon user logging in they will now see a Category nav link. Upon clicking Category nav link, user will be redirected to a list view of categories ordered alphabetically. Upon logging out user will be redirected to Login page and the categories nav link or list items will not longer appear. 

Changes made: 
Changes made to Application Views, Header.js, App.js, (specific changes made to the logout menu item to redirect user to login upon click affordance. Model, repository, and controller added for categories. 

Testing Instructions: 
Git fetch --all
git checkout 'my branch'
make sure to run both your server and client side apps with data in your API
log in, click on nav link, log out.
